src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -63,62 +63,62 @@ components:
         issueDate:
           type: string
           format: date
-          description: The issue date of the Voter Authority Card
+          description: The issue date of the Voter Authority Certificate
           example: '2022-06-01'
         requestDateTime:
           type: string
           format: date-time
-          description: The date time the Voter Authority Certificate was requested
+          description: The date time the Voter Authority Certificate print request was requested
           example: '2022-06-01T12:23:03.000Z'
-        cardFirstname:
+        firstName:
```

Comment:
@matt-wills-valtech @Neil-Massey ... "firstName", "middleNames", "surname" ... are these property names consistent with similar messages / fields in RCA and notifications-api ?

Consistency is a good thing 😁 👍

Reply:
Notifications API doesn't define these (the template personalisation allows the placeholder name and value to be provided by the caller)

Reply:
I have checked in VCA wrt what it sends to RCA, and its is "firstName", "middleNames" and "surname"; so we are consistent 👍